src/core/api/index.js

```diff
@@ -62,27 +63,35 @@ export function callApi({
 
   return fetch(apiURL, options)
     .then((response) => {
-      return response.clone().json()
-        .then(
-          (jsonResponse) => ({ response, jsonResponse }),
-          (error) => {
+      function makeTextResponseJson(textResponse) {
+        return { jsonResponse: { text: textResponse }, response };
+      }
+
+      if (response.headers.get('Content-Type') === 'application/json') {
```

Comment:
So the "conten-type" header isn't special but MDN was out of date. I checked the spec and Firefox and Chrome matched the spec. I updated MDN.